app/controllers/api/v1/admin/user/goals_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::V1::Admin::User::GoalsController < Api::V1::BaseController
```

Comment:
you've created resources in scope of user, you are loading user, but you do nothing with that.

Reply:
Also, please consider to move this under evaluations.